### DISCUSSION TITLE
What do we want from LNUG as a *remote* meetup in 2021?

### DISCUSSION THREAD

**Discussion by admataz** [lnug/feedback#120] [General]
I've found there are challenges to running a regular tech event that is 100% online without benefits and buzz of real humans, pizza and beer in the same room. I was that much disappointed with the experience of trying to replicate our  online meetups as video calls, that I gave up after June and took  a nice long break to think about it.  

This lockdown status is going to last a while, so I'm ready to head back into 2021 with a remote-first approach, and thinking about how we adapt LNUG activities around this. 

Let's discuss our plans for 2021. How do you see it working best? 

cc @lnug/organisers @lnug/contributors

**Comment by seanmtracey** [comment]
Maybe instead of having a fixed online meetup, we could offer support to people who would like to give a talk, put it up on the LNUG YouTube, and then offer a time (say a week later) that people could join a call for a Q+A with the speaker - or maybe a panel that the organisers could host with the speaker?

Would be more async than a traditional online meetup, but it could give people time to absorb the content at their convenience and think about what kind of discussion they'd like to have with the content creator?

**Comment by hewIngram** [comment]
Yeah I'm curious if we can shimmy to more of a content creation thing, we could quite easily record and release talks which feels better than trying to make an online meetup happen and a better way of making more long lasting content.
I think that's a good shout Sean.

**Comment by phazonoverload** [comment]
I’m going to come with the exact opposite suggestion - I think there’s enough content in the world that people’s backlogs of things to learn is plenty full. 

Instead, I’d love to focus more on utilising tech to facilitate conversations between people. I think there’s been some excellent stabs at online networking this year, but nothing I’ve tried has quite hit the spot. 

Meetups, for me at least, are more about chatting to people I know share a common interest than the content put on by organisers.

**Comment by hewIngram** [comment]
Yeah that's an interesting point. So the plan @phazonoverload atm is to try to get some kind of discord server going and I *think* that is a fairly elegant way to try to do those things but I think we need to do a bunch of legwork to make it successful. Would love your thoughts - I'll ping you an invite

**Comment by admataz** [comment]
This is a great discussion for the challenges we were facing - but it's time to close it. Hopefully we aren't forced back to remote meetups any time.
